### ISSUE TITLE
Create two instances of database using Dockerfile. 

### ISSUE THREAD

**Issue by payalgaikwad42** [docker/hub-feedback#1839]
Hi, 
I have a dockerfile which creates the postgres container instance. 

> 
FROM postgres 
CMD ["postgres"]

I have environment variable set in deployment.json file as follows:

> "db": {
            "version": "1.0",
            "type": "docker",
            "status": "running",
            "restartPolicy": "always",
            "settings": {
              "image": "${MODULES.db}",
              "createOptions": {
                "Env": [
                  "POSTGRES_PASSWORD=postgres"
                ],
                "HostConfig": {
                  "PortBindings": {
                    "5432/tcp": [
                      {
                        "HostPort": "5432"
                      }
                    ]
                  },
                  "Mounts": [
                    {
                      "Target": "/etc/timezone",
                      "Source": "/etc/timezone",
                      "Type": "bind"
                    },
                    {
                      "Target": "/etc/localtime",
                      "Source": "/etc/localtime",
                      "Type": "bind"
                    },
                    {
                      "Target": "/var/lib/postgresql/data/",
                      "Source": "/var/local/",
                      "Type": "bind"
                    }
                  ]
                }
              }
            }
        }

I am creating a docker image from this and deployed it as a iotedge module.
I am using sequelize ORM to populate this database by taking the data from backend using api call.

My question is can I create two instances of database inside same docker container one is getting populated using sequelize orm and another one is getting populated using some code in dotnet core. 

so basically I want one container with two databases running inside it getting populated using two different techniques. 

Can any one give some comments on this?

**Comment by cowsrule** [comment]
This repository is for Docker Hub specific feedback. Please visit https://forums.docker.com/ to discuss your question.
